### PR TITLE
bugfix/13876-currentPriceIndicator-stockTools-error

### LINKED ISSUE #13876: Disabling current price indicator via stock tools (yAxis.crosshairs.label + lastVisiblePrice) error
#### Expected behaviour
No error.

#### Actual behaviour
When the `yAxis.crosshair.label` and `lastVisiblePrice` are enabled after disabling the current price indicator vis stock tools button the following error will occur: `Uncaught TypeError: Cannot read property 'setAttribute' of undefined`.

It might be related to https://github.com/highcharts/highcharts/issues/13875

#### Live demo with steps to reproduce
Steps: 
1/ Hover over the plot area
2/ Click on the 'Current Price Indicator` button 3 times
3/ Check error in the console.

https://jsfiddle.net/BlackLabel/rt5wgczo/

#### Product version
8.1.2

#### Affected browser(s)
All

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed #13876, toggling `currentPriceIndicator` button threw error.

### CODE REVIEW

**Review by raf18seb — CHANGES_REQUESTED** [review]

File: samples/unit-tests/indicator-price-indicator/recalculations/demo.js [hunk 1]
```diff
@@ -147,3 +147,142 @@ QUnit.test(
...
+        chart.yAxis[0].crossLabel.visibility,
+        'hidden',
+        'Cross label should not be visible.'
+    );
+
+    // Show currentPriceIndicator with the label.
+    chart.navigationBindings.options.bindings.currentPriceIndicator.init.call(
+        chart.navigationBindings, button);
+    assert.strictEqual(
+        chart.series[0].crossLabel.visibility,
```
[the remark this answers is not in the document]
Comment (pawelfus) [reply samples/unit-tests/indicator-price-indicator/recalculations/demo.js:213]: In source code, yes, please. In tests, it's not that important, but we may as well practice this :)

File: samples/unit-tests/indicator-price-indicator/recalculations/demo.js [hunk 2]
```diff
@@ -147,3 +147,142 @@ QUnit.test(
...
+    const chart = Highcharts.stockChart('container', {
+            stockTools: {
+                gui: {
+                    enabled: true,
+                    buttons: ['currentPriceIndicator']
+                }
+            },
+            yAxis: [{
+                crosshair: {
+                    color: 'blue',
```
[the remark this answers is not in the document]
Comment (karolkolodziej) [reply samples/unit-tests/indicator-price-indicator/recalculations/demo.js:163]: I changed most of them, the only one left is the `red` for the `lastPrice` where it looks like a separate issue  #15074

File: samples/unit-tests/indicator-price-indicator/recalculations/demo.js [hunk 3]
```diff
@@ -147,3 +147,142 @@ QUnit.test(
...
     }
 );
+
+QUnit.test(`The currentPriceIndicator, lastPrice and axis crosshair should
+    have their own properties, no errors in console after toggle, #13876.`,
+function (assert) {
+    const chart = Highcharts.stockChart('container', {
+            stockTools: {
+                gui: {
+                    enabled: true,
```
[the remark this answers is not in the document]
Comment (raf18seb) [reply samples/unit-tests/indicator-price-indicator/recalculations/demo.js:157]: Sure, in that case you can leave it as it is!

**Review by raf18seb — APPROVED** [review]
Thanks!

**Review by pawelfus — APPROVED** [review]
Looks good!

**Review by pawellysy — APPROVED** [review]
Looks Good!

File: samples/unit-tests/indicator-price-indicator/recalculations/demo.js [hunk 4]
```diff
@@ -147,3 +147,142 @@ QUnit.test(
         );
     }
 );
+
+QUnit.test(`The currentPriceIndicator, lastPrice and axis crosshair should
+    have their own properties, no errors in console after toggle, #13876.`,
```
[the remark this answers is not in the document]
Comment (raf18seb) [reply samples/unit-tests/indicator-price-indicator/recalculations/demo.js]: @pawellysy yes, we were talking about descriptions in **asserts**:
```js
assert.strictEqual(
    1 + 2,
    3,
    '1 + 2 should equal 3.'
);
```

In the above comment, I was talking about the **main title**. In the main title, you should shortly describe what you are testing or what was the main issue.

Please, have a look at this test for a reference: https://github.com/highcharts/highcharts/blob/master/samples/unit-tests/series-dumbbell/markers/demo.js
You will find 2 separate main tests with titles describing what the tests are about. Then, in every test, you will find many asserts testing a specific case with a "should" keyword indicating how the result should look like.

**Pull request merged**